### ISSUE TITLE
quantized batchnorm parameters/buffers not saved in state_dict

### ISSUE THREAD

**Issue by jerryzh168** [pytorch/pytorch#69808]
### 🐛 Describe the bug

Reported from https://discuss.pytorch.org/t/quantized-batch-norm-parameters-not-saved-in-state-dict/138227, probably need to do a similar fix like: https://github.com/pytorch/pytorch/blob/master/torch/nn/quantized/modules/activation.py#L102 for quantized batchnorm module

### Versions

pytorch master

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo

**Comment by vkuzo** [comment]
we are tracking this in https://github.com/pytorch/pytorch/issues/43774
